x/mongo/driver/description/server.go

```diff
@@ -390,3 +390,9 @@ func decodeStringMap(element bsoncore.Element, name string) (map[string]string,
 	}
 	return m, nil
 }
+
+// SupportsRetryWrites returns true if retryable writes are supported which requires
```

Comment:
nit: I appreciate the documentation, but docs and comments should usually focus on "what" and "why" rather than "how". The fact that this checks the value of `logicalSessionTimeoutMinutes` and the server kind are obvious from the function's contents. I'd change this to something like

```
SupportsRetryWrites returns true if this description represents a server that supports retryable writes.
```